### PR TITLE
Update notifications.mdx

### PR DESCRIPTION
**Pull request by KennyAtHPE**
[Preview](https://deploy-preview-1713--keen-mayer-a86c8b.netlify.app/templates/notifications)

#### What does this PR do?
Included status updates and resources for notifications.

### CODE REVIEW

File: aries-site/src/pages/templates/notifications.mdx [hunk 1]
```diff
@@ -1 +1,25 @@
-More content coming soon.
+## Status
+
+Notifications is currently on the exploration and research phase.
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:3]:
```suggestion
The Design System team is currently exploring and researching best practices and considerations for notification patterns. To stay up-to-date with the work being done, check out the links below:
```

File: aries-site/src/pages/templates/notifications.mdx [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+Stay in the loop and check out what is being worked on now:
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:5]:
```suggestion
```

File: aries-site/src/pages/templates/notifications.mdx [hunk 3]
```diff
@@ -1 +1,25 @@
...
+
+* Defining the taxonomy
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:7]: Can we link to specific frame of Figma documents where possible?

File: aries-site/src/pages/templates/notifications.mdx [hunk 4]
```diff
@@ -1 +1,25 @@
...
+* Documenting accessibility guidelines
+* Identifying potential user research opportunities
+* Researching and reviewing literature and other external resources
+
+### Resources
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:12]:
```suggestion
### Additional Resources
```

File: aries-site/src/pages/templates/notifications.mdx [hunk 5]
```diff
@@ -1 +1,25 @@
...
+
+Aside from what the [design system](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, be sure to view the work done by other teams on Notifications:
+
+* [Header Footer Service (HFS)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059) 
+* [Unified Customer Experience Platform (UCEP)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059)
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:17]: This link needs to be updated to the UCEP link

File: aries-site/src/pages/templates/notifications.mdx [hunk 6]
```diff
@@ -1 +1,25 @@
...
+* [Common Cloud Service (CCS)](https://www.figma.com/file/vT3CaJANRAKHysuZs2JIGR/CCS-Notifications?node-id=1%3A17948)
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:18]: Can we put this link first?

File: aries-site/src/pages/templates/notifications.mdx [hunk 7]
```diff
@@ -1 +1,25 @@
...
+Stay in the loop and check out what is being worked on now:
+
+* Defining the taxonomy
+* Documenting accessibility guidelines
+* Identifying potential user research opportunities
+* Researching and reviewing literature and other external resources
+
+### Resources
+
+Aside from what the [design system](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, be sure to view the work done by other teams on Notifications:
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:14]:
```suggestion
In additional to the exploration that the [HPE Design System](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, you may find it helpful to view the research done by other teams on Notifications.
```

**Review by ericsoderberghp — APPROVED** [review]

File: aries-site/src/pages/templates/notifications.mdx [hunk 8]
```diff
@@ -1 +1,21 @@
...
+
+In additional to the exploration that the [HPE Design System](https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=0%3A1) has done, you may find it helpful to view the research done by other teams on Notifications.
+
+* [Common Cloud Service (CCS)](https://www.figma.com/file/vT3CaJANRAKHysuZs2JIGR/CCS-Notifications?node-id=1%3A17948)
+* [Header Footer Service (HFS)](https://www.figma.com/file/AbxAC3KU59pPyHG5mnnh7R/HFS-North-Star?node-id=140%3A10059) 
+* [Unified Customer Experience Platform (UCEP)](https://app.mural.co/t/hpecdoresearchteam7715/m/hpecdoresearchteam7715/1622038532199/60894a815784273763b6c9c3d71df4dc38fa8cb0?sender=uaae76a8060cc23bc06005320)
+
+### Have more questions?
+
+Please be sure to send them over to the #hpe-design-system channel on slack!
```
Comment (taysea) [inline aries-site/src/pages/templates/notifications.mdx:21]:
```suggestion
Please be sure to send them over to the `#hpe-design-system` channel on [Slack](https://slack-invite.grommet.io/)!
```

**Review by taysea — APPROVED** [review]
Nice job!

**Pull request merged**